services/movie_session.py

```diff
@@ -0,0 +1,55 @@
+from django.utils.datetime_safe import datetime
+
+from db.models import MovieSession, Movie, CinemaHall
+from django.utils.dateparse import parse_date
+
+
+def create_movie_session(
+        movie_show_time: datetime,
+        movie_id: int,
+        cinema_hall_id: int
+) -> MovieSession:
+    movie = Movie.objects.get(id=movie_id)
+    cinema_hall = CinemaHall.objects.get(id=cinema_hall_id)
+    return MovieSession.objects.create(
+        show_time=movie_show_time,
+        movie=movie,
+        cinema_hall=cinema_hall
+    )
+
+
+def get_movies_sessions(session_date: datetime = None) -> MovieSession:

```

Comment:
The return type hint for `get_movies_sessions` should be `QuerySet[MovieSession]` instead of `MovieSession`, as the function returns a QuerySet containing multiple `MovieSession` instances.